modules/administration/pages/reports.adoc

```diff
@@ -0,0 +1,89 @@
+
+You can generate reports on:
+
+. System Inventory: list all the systems registered to {productname}.
+. Patches: list all the patches relevant to the registered systems and sorts patches by severity, as well as the systems that apply to a particular patch.
+. Users: list all registered users and any systems associated with a particular user.
+
+To get the report in CSV format, run this command at the command prompt on the server:
+
+----
+spacewalk-report <report_name>
+----
+
+This table lists the available reports:
+
+
+[[tab.bp.troubleshooting.spacewalk-report]]
+.[command]``spacewalk-report`` Reports
+[cols="1,1,1", options="header"]
+|===
+|Report | Invoked as | Description
+| Actions | [command]``actions`` | All actions.
+| Activation Keys | [command]``activation-keys`` | All activation keys, and the entitlements, channels, configuration channels, server groups, and packages associated with them.
+| Activation Keys: Channels | [command]``activation-keys-channels`` | All activation keys and the entities associated with each key.
+| Activation Keys: Configuration | [command]``activation-keys-config`` | All activation keys and the configuration channels associated with each key.
+| Activation Keys: Server Groups | [command]``activation-keys-groups`` | All activation keys and the server groups associated with each key.
+| Activation Keys: Packages | [command]``activation-keys-packages`` | All activation keys and the packages each key can deploy.
+| Channel Packages | [command]``channel-packages`` | All packages in a channel.
+| Channel Report | [command]``channels`` | Detailed report of a given channel.
+| Cloned Channel Report | [command]``cloned-channels`` | Detailed report of cloned channels.
+| Configuration Files | [command]``config-files`` | All configuration file revisions for all organizations, including  file contents and file information.
+| Latest Configuration Files | [command]``config-files-latest`` | The most recent configuration file revisions for all organizations, including  file contents and file information.
+| Custom Channels | [command]``custom-channels`` | Channel-meta-data for all channels owned by specific organizations
+| Custom Info | [command]``custom-info`` | System custom information.
+| Entitlements | [command]``entitlements`` | All organizations, and their system or channel entitlements.
+| Patches in Channels | [command]``errata-channels`` | All patches in channels.
+| Patches Details | [command]``errata-list`` | All patches that affect registered systems.
+| All patches | [command]``errata-list-all`` | All patches.
+| Patches for Systems | [command]``errata-systems`` | Applicable patches and any registered systems that are affected.
+| Host Guests | [command]``host-guests`` | Host and guests mapping.
+| Inactive Systems | [command]``inactive-systems`` | Inactive systems.
```

Comment:
```suggestion
| Inactive Clients | [command]``inactive-systems`` | Inactive clients.
```

I am actually a bit unsure about this and the following ones, as those are the titles of the pages in the Web UI that have the same information. Up to you to decide I'd say.

Reply:
I really much prefer "clients" over "systems", because "systems" could mean just about anything. I get the point, but considering the report name remains "systems" I reckon people will figure it out.